src/name_bazaar/server/deployer.cljs

```diff
@@ -30,11 +30,22 @@
   (deploy-smart-contract! :ens (merge default-opts {:gas 700000})))
 
 
-(defn deploy-registrar! [default-opts]
+(defn deploy-resolvers! [default-opts]
+  (deploy-smart-contract! :reverse-name-resolver (merge default-opts {:gas 3200000}))
+  (deploy-smart-contract! :public-resolver (merge default-opts
+                                                    {:gas 3200000
+                                                     :arguments [(contract-address :ens)]})))
+
+
+(defn deploy-registrars! [default-opts]
   (deploy-smart-contract! :name-bazaar-registrar (merge default-opts
```

Comment:
`:name-bazaar-registrar` should really be called `eth-registrar` everywhere, even in derived function names. It was called `:registrar` when we started, but we managed to rename it wrong :)

Reply:
True, lets rename it everywhere in code.